grammars/asciidoc.cson

```diff
@@ -303,7 +303,7 @@
       #
       #   *bold phrase*
       {
-        'match': '(?<=^|\\s)(\\*[^\\*]+\\*)'
+        'match': '(?<=^|\\s)(\\*[^\\s^\\*][^\\*]*\\*)(?=\\s|$)'
```

Comment:
I think the final group should be:

```
(?= |\\t|$)
```

We want to match spaces and tabs but not other forms of whitespace. Technically, the result is probably the same since as far as understand the grammar matches per line, but this communicates the intent better.